_docs/_latest/develop/index.md

```diff
@@ -19,6 +19,10 @@ a pull request.
 Understand how Forseti uses branches, performs releases, and manages development of Forseti on
 GitHub.
 
+**[Edit & Build Forseti Security Website Locally]({% link _docs/latest/develop/website.md %})**
```

Comment:
Can you please move this section to under the `Start Developing on Forseti` as that is likely what folks would do first, so we should order that more appropriately higher up.

Also, can you rename this section to `Install, Buiild, and Run Forseti Website.`